docs/notebooks/Poincare Evaluation.ipynb

```diff
@@ -0,0 +1,1448 @@
+    "A more detailed explanation of the tasks and the evaluation methodology is present in the individual evaluation subsections."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Setup\n",
+    "\n",
+    "The following code clones the `poincare-embedding` repository containing the C++ implementation of the Poincare embeddings, and applies a patch containing minor additions to the implementation. Please set the variable `parent_directory` below to define the directory to which the repository is cloned."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set this to the path of the directory that is to contain the poincare-embedding directory\n",
+    "parent_directory = '/home/jayant/projects/'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "current_directory = os.getcwd()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/home/jayant/projects\n",
+      "Cloning into 'poincare-embedding'...\n",
+      "remote: Counting objects: 96, done.\u001b[K\n",
+      "remote: Total 96 (delta 0), reused 0 (delta 0), pack-reused 96\u001b[K\n",
+      "Unpacking objects: 100% (96/96), done.\n",
+      "Checking connectivity... done.\n",
+      "/home/jayant/projects/poincare-embedding\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Clone repo\n",
+    "% cd {parent_directory}\n",
+    "repo_name = 'poincare-embedding'\n",
+    "! git clone https://github.com/TatsuyaShirakawa/poincare-embedding.git {repo_name}\n",
+    "% cd {repo_name}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2017-10-18 20:37:11--  https://raw.githubusercontent.com/RaRe-Technologies/gensim/poincare_eval/docs/notebooks/poincare_data/poincare_burn_in_eps.patch\n",
+      "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.112.133\n",
+      "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.112.133|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 7999 (7.8K) [text/plain]\n",
+      "Saving to: ‘./poincare_burn_in_eps.patch’\n",
+      "\n",
+      "./poincare_burn_in_ 100%[===================>]   7.81K  --.-KB/s    in 0s      \n",
+      "\n",
+      "2017-10-18 20:37:11 (59.5 MB/s) - ‘./poincare_burn_in_eps.patch’ saved [7999/7999]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Download and apply patch\n",
+    "patch_url = \"https://raw.githubusercontent.com/RaRe-Technologies/gensim/poincare_eval/docs/notebooks/poincare_data/poincare_burn_in_eps.patch\"\n",
```

Comment:
The patch is in the repo so you can just reference the local file (`poincare_data/poincare_burn_in_eps.patch`). Besides, the URL will get broken once the code is merged to master.

Reply:
Good point, changed.